Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -874,8 +874,6 @@ PhysicalParticleContainer::FieldGather (int lev,
                                         const amrex::MultiFab& By,
                                         const amrex::MultiFab& Bz)
 {
-    const std::array<Real,3>& dx = WarpX::CellSize(lev);
-
```

Comment:
Fixes the warning:
`./Source/Particles/PhysicalParticleContainer.cpp:861:31`: warning: unused variable ‘dx’ [-Wunused-variable]
     const std::array<Real,3>& dx = WarpX::CellSize(lev);